substrate/frame/scheduler/src/benchmarking.rs

```diff
@@ -92,8 +89,9 @@ fn make_task<T: Config>(
 }
 
 fn bounded<T: Config>(len: u32) -> Option<BoundedCallOf<T>> {
-	let call =
-		<<T as Config>::RuntimeCall>::from(SystemCall::remark { remark: vec![0; len as usize] });
+	let call = <<T as Config>::RuntimeCall>::from(frame_system::Call::remark {
```

Comment:
Why did you change `SystemCall` to `frame_system::Call`? I find the old one as readable as the new one, but more compact.